core/domain/app_feedback_report_constants.py

```diff
@@ -51,24 +52,75 @@
 STATS_ID_DELIMITER = ':'
 ANDROID_VERSION_NAME_DELIMITER = '-'
 
-# Ignoring the untyped call error because python_utils is untyped.
-REPORT_TYPE = python_utils.create_enum('suggestion', 'issue', 'crash') # type: ignore[no-untyped-call]
-CATEGORY = python_utils.create_enum( # type: ignore[no-untyped-call]
-    'feature_suggestion', 'language_suggestion', 'other_suggestion',
-    'lesson_question_issue', 'language_general_issue', 'language_audio_issue',
-    'language_text_issue', 'topics_issue', 'profile_issue', 'other_issue',
-    'lesson_player_crash', 'practice_questions_crash', 'options_page_crash',
-    'profile_page_crash', 'other_crash')
-ENTRY_POINT = python_utils.create_enum( # type: ignore[no-untyped-call]
-    'navigation_drawer', 'lesson_player', 'revision_card', 'crash')
-STATS_PARAMETER_NAMES = python_utils.create_enum( # type: ignore[no-untyped-call]
-    'platform', 'report_type', 'country_locale_code',
-    'entry_point_name', 'text_language_code', 'audio_language_code',
-    'android_sdk_version', 'version_name')
-ANDROID_TEXT_SIZE = python_utils.create_enum( # type: ignore[no-untyped-call]
-    'text_size_unspecified', 'small_text_size', 'medium_text_size',
-    'large_text_size', 'extra_large_text_size')
-ANDROID_NETWORK_TYPE = python_utils.create_enum('wifi', 'cellular', 'none') # type: ignore[no-untyped-call]
+
+class REPORT_TYPE(enum.Enum): # pylint: disable=invalid-name
+    """Enum for report types."""
+
+    suggestion = 'suggestion' # pylint: disable=invalid-name
+    issue = 'issue' # pylint: disable=invalid-name
+    crash = 'crash' # pylint: disable=invalid-name
```

Comment:
Hey, @seanlip and @DubeySandeep how do you feel towards making our enums more like they are in other codebases and also more pylint compliant, that is using SnakeCase for the enum name and then UPPERCASE for the values? From my understanding, this will not even influence the data in our storages since the values can stay in lowercase.

Reply:
Hi @vojtechjelinek -- just to check that I understand the proposal correctly: this is snake_case, but the way you've written it seems like UpperCamelCase. What should it be?

If you could provide a full example of each of the different alternatives to choose from, that would probably be helpful.

Reply:
The current approach is like this
```python
class REPORT_TYPE(enum.Enum): # pylint: disable=invalid-name
    """Enum for report types."""

    suggestion = 'suggestion' # pylint: disable=invalid-name
    issue = 'issue' # pylint: disable=invalid-name
    crash = 'crash' # pylint: disable=invalid-name

report_type = REPORT_TYPE.suggestion
```

My suggested approach and the one that is in [Python docs](https://docs.python.org/3/library/enum.html) is like this:
```python
class ReportType(enum.Enum):
    """Enum for report types."""

    SUGGESTION = 'suggestion'
    ISSUE = 'issue'
    CRASH = 'crash'

report_type = ReportType.SUGGESTION
```

Reply:
Thanks! 

For my part, I'm totally fine with either -- let's just make sure we're consistent throughout the codebase, document it in the [coding style guide](https://github.com/oppia/oppia/wiki/Coding-style-guide), and have lint checks that enforce it. The point about it not affecting datastore values is an important one -- that's a good thing to verify.

Reply:
@JeeveshGarg Can you please change the formatting of all the enums to the suggested style (even for existing enums that are in the class format already)? I will later verify your changes on the backup server just to make sure that everything works correctly.

Reply:
@vojtechjelinek i changed the format of enum of every test file successfully, but when i try to change in rest files, encounter lots of backend tests error for each file, because enum class uses in other files as well. I change them also and trying to solve them also, but seems too complicated change all togeather and one enum class `NAMES` in models.py is used in 264 files.

can i add TODO comment for that and cover each one separately in one new issue?

Reply:
Done